<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>custom</class>
 <widget class="QWidget" name="custom">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Purisa</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border: 5px solid;
	border-radius: 15px;
	background-color: rgb(79, 255, 53);
	font: 16pt &quot;Purisa&quot;;
}
QPushButton::hover{
	background-color: red;
}
QPushButton::pressed{
	background-color: red;
	border: 10px solid green;
}
QLabel{
font: 16pt &quot;Purisa&quot;;
}
QComboBox{
font: 16pt &quot;Purisa&quot;;
}
QToolButton{
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>671</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Fire</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_7">
      <property name="styleSheet">
       <string notr="true">border: 2px solid green;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>Player 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Purisa</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="combo_up">
      <property name="styleSheet">
       <string notr="true">border: 2px solid green;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>UP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOWN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LEFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>w</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>e</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>r</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>t</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>u</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>i</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>o</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>j</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>k</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>l</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LSHIFT</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="combo_down">
      <property name="styleSheet">
       <string notr="true">border: 2px solid green;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>UP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOWN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LEFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>w</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>e</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>r</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>t</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>u</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>i</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>o</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>j</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>k</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>l</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LSHIFT</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="combo_left">
      <property name="styleSheet">
       <string notr="true">border: 2px solid green;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>UP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOWN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LEFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>w</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>e</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>r</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>t</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>u</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>i</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>o</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>j</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>k</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>l</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LSHIFT</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="combo_right">
      <property name="styleSheet">
       <string notr="true">border: 2px solid green;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>UP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOWN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LEFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>w</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>e</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>r</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>t</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>u</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>i</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>o</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>j</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>k</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>l</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LSHIFT</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="combo_fire">
      <property name="styleSheet">
       <string notr="true">border: 2px solid green;
border-radius: 3px;</string>
      </property>
      <item>
       <property name="text">
        <string>UP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOWN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LEFT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>w</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>e</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>r</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>t</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>u</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>i</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>o</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>d</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>g</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>j</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>k</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>l</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>b</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>c</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RCTRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RALT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LSHIFT</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="bt_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_feed">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_save">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
